php/composer.json

```diff
@@ -1,7 +1,31 @@
 {
-    "name": "dzcode.io/kuliya",
+    "name": "dzcodeio/kuliya",
     "description": "Algeria's college hierarchy dataset as a package.",
+    "keywords": [
+        "dzcode", "algeria", "dataset", "package", "college"
+    ],
+    "license": "MIT",
+    "authors": [
```

Comment:
Put your information, since you are the author of the package

Reply:
done👌